Code/Tools/SerializeContextTools/Runner.cpp

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
The entirety of the contents of this file was moved from main.cpp.
There are no code changes within this file.